src/runtime/substitution.cc

```diff
@@ -18,6 +18,7 @@
 #include "flexflow/ffconst_utils.h"
 #include "flexflow/graph.h"
 #include "flexflow/graph_structures.h"
+#include "flexflow/operator.h"
```

Comment:
Why?

Reply:
Maybe this is not needed. I tried and make -j 15 will fail but make will be okay.

I have removed it and let the CI to check again.